component/navbar/navbar.css

```diff
@@ -0,0 +1,36 @@
+.navbar-component{
+    display: flex;
+    background-color: var(--black);
```

Comment:
Just a suggestion, use a different background color for the navbar as its difficult to see it

Reply:
fixed it. thanks